index.js

```diff
@@ -23,6 +23,9 @@ async function purgePage(bot, title){
 /**
 *   Function that refreshes the Dynamic Page List the bot uses by performing a null edition.
 *   Technically we could purge the cache here, but by hardcoding the contents of page we've got an additional layer of protection agains vandals :)
+*
+*   NOTE: THIS FUNCTION IS NO LONGER USED, DPL PAGES SHOULD BE PROTECTED.

```

Comment:
Consider removing the function eventually. It's still available in the git history, in case it's needed in future.